recipe/meta.yaml

```diff
@@ -1,43 +1,46 @@
 {% set name = "tensorflow-metadata" %}
-{% set version = "1.15.0" %}
-
+{% set version = "1.16.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/py3/t/tensorflow-metadata/tensorflow_metadata-{{ version }}-py3-none-any.whl
-  sha256: cb84d8e159128aeae7b3f6013ccd7969c69d2e6d1a7b255dbfa6f5344d962986
+  url: https://pypi.org/packages/py3/t/tensorflow-metadata/tensorflow_metadata-{{ version }}-py3-none-any.whl
```

Comment:
Is it possible to build this from the source instead of the wheel. 

https://github.com/tensorflow/metadata

Reply:
@hmaarrfk 
It would add Bazel build plus multitudes of patches.
Sure, it would be ideal if the source was used, but the end result is same and it removes a lot of headache (Bazel is only used to generate platform-agnostic pure Python files).

Reply:
I'm not ok, but i'll let the original maintainers review and you can all decide.

bye from here!

Reply:
I personally think that using Bazel here to generate pure Python code is wasteful, and this was a decision made before this PR.
But whichever way, I'll wait for other contributors for their say. Thank you for your opinion.

Reply:
> doesn't seem this was ever built from source...

Yeah, it will be painful without added value, I also believe so.